src/interfaces/store-interfaces/state-interfaces/business-id-interface.ts

```diff
@@ -0,0 +1,37 @@
+import { EntityState, CorpTypeCd } from '@/enums'
+import { IsoDatePacific, ApiDateTimeUtc } from '@bcrs-shared-components/interfaces'
+
+/** The Business Warning object. */
+export interface BusinessWarningIF {
+  code: string // FUTURE: use an enum
+  message: string
+  warningType: string
+  filing: string
+}
+
+/** The Business object from the API. */
+export interface BusinessIdIF {
```

Comment:
I think we should have a single Business Interface. Could this object easily be merged with the existing BusinessIF?

Reply:
Thanks. Looks good.